docs/src/main/asciidoc/qute.adoc

```diff
@@ -26,45 +26,48 @@ Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {q
 
 The solution is located in the `qute-quickstart` link:{quickstarts-tree-url}/qute-quickstart[directory].
 
-== Hello World with Jakarta REST
-
-If you want to use Qute in your Jakarta REST application, you need to add an extension first:
+[[hello-qute-web]]
+== Hello World with Qute Web
 
-* either `quarkus-rest-qute` if you are using Quarkus REST (formerly RESTEasy Reactive):
-+
+If you want to use Qute in your Quarkus Web application, add the Qute Web extension first:
 [source,xml,role="primary asciidoc-tabs-target-sync-cli asciidoc-tabs-target-sync-maven"]
 .pom.xml
 ----
 <dependency>
-    <groupId>io.quarkus</groupId>
-    <artifactId>quarkus-rest-qute</artifactId>
+    <groupId>io.quarkiverse.qute.web</groupId>
+    <artifactId>quarkus-qute-web</artifactId>
 </dependency>
 ----
 +
 [source,gradle,role="secondary asciidoc-tabs-target-sync-gradle"]
 .build.gradle
 ----
-implementation("io.quarkus:quarkus-rest-qute")
+implementation("io.quarkiverse.qute.web:quarkus-qute-web")
 ----
 
-* or `quarkus-resteasy-qute` if you are using RESTEasy Classic:
-+
-[source,xml,role="primary asciidoc-tabs-target-sync-cli asciidoc-tabs-target-sync-maven"]
-.pom.xml
-----
-<dependency>
-    <groupId>io.quarkus</groupId>
-    <artifactId>quarkus-resteasy-qute</artifactId>
-</dependency>
-----
-+
-[source,gradle,role="secondary asciidoc-tabs-target-sync-gradle"]
-.build.gradle
+All files located in the `src/main/resources/templates` directory and its subdirectories are registered as templates. Templates are validated during startup and watched for changes in the development mode.
+
+Now, let's start with a Hello World html template:
+
+.src/main/resources/templates/pub/hello.html
+[source]
 ----
-implementation("io.quarkus:quarkus-resteasy-qute")
+<h1>Hello {http:param('name', 'Quarkus')}!</h1> <1>
 ----
+<1> `{http:param('name', 'Quarkus')}` is an expression that is evaluated when the template is rendered (Quarkus is the default value).
 
-We'll start with a very simple template:
+NOTE: Templates located in the `pub` directory are served via HTTP. Automatically, no controllers needed. For example, the template src/main/resource/templates/pub/foo.html will be served from the paths /foo and /foo.html by default.
+
+If your application is running, you can open your browser and hit:  http://localhost:8080/hello?name=Martin
+
+For more information about Qute Web, see the https://docs.quarkiverse.io/quarkus-qute-web/dev/index.html[Qute Web guide].
+
+[[hello-qute-rest]]
+== Hello World with Jakarta REST
+
+If you want to use Qute in your Jakarta REST application, you still need to add the Qute Web extension first (see <<hello-qute-web>>) and make sure you have the Quarkus REST (formerly RESTEasy Reactive) extension.
```

Comment:
Qute Web is part of the platform, this has already been discussed

Reply:
@ia3andy it does reads problematically here though - it would be good to somehow capture what the discussion was and be able to answer gsmet's question why it was not added to quarkus-rest.

Reply:
The idea is:
- Qute Web + REST = Qute REST
- Qute Web + RESTEasy = Qute RESTEasy

This is the concept of conditional dependency to avoid having hundreds of listed extensions just for compatibly. I am not sure if it's already configured, but it should be the same for Jackson or other case like this.
- REST + Jackson = REST Jackson (no need to add it)

Reply:
Still my problem is that we are pushing something that is not supported (yet?) as the way to go whereas there is a perfectly supported alternative that is working and has always worked.
Being in the Platform does not mean being supported.

Not everyone is (unfortunately) comfortable with using Quarkiverse extensions.

And while I don't have issues with adding info about Quarkiverse extensions here, I think it's a bit unfortunate we drop some perfectly working and supported way of doing things.

Now that's @maxandersen call here. But I have the impression you're pushing this a bit too hard on everyone.

Reply:
@ia3andy you have to remind me again why is it that we ended up doing the conditional dependency trick here outside in quarkiverse and not in Quarkus core?

@gsmet the platform is more about what is supported than anything else we have. we got stuff in core that is most definitely less supported :) i do agree we should have this clarified since:

a) if we can make it work better by just having the qute res stuff in core we probably should and we can remove this unfortuante mention about you must use quarkiverse for jakarta rest)
b) if we can't - we should rephrase/move this around to be more of a recommendation/suggestion.